### ISSUE TITLE
block this domain

### ISSUE THREAD

**Issue by holequantum** [badmojr/1Hosts#595]
`graph.instagram.com`
mostly used in app with trackers as instagram.
this does collect analytics in instagram blocking it does not effect instagram

**Comment by holequantum** [comment]
`viapplogs.net` this domain also it collects app usage logs

**Comment by badmojr** [comment]
> graph.instagram.com

Blocked on **Pro**! Please upgrade.

**Comment by badmojr** [comment]
> `viapplogs.net` this domain also it collects app usage logs

Don't do this please.
Open a separate issue instead.

**Comment by holequantum** [comment]
sure sorry about that.
